DisplaySettings/CHANGELOG.md

```diff
@@ -16,6 +16,10 @@ All notable changes to this RDK Service will be documented in this file.
 
 * For more details, refer to [versioning](https://github.com/rdkcentral/rdkservices#versioning) section under Main README.
 
+## [1.0.7]
```

Comment:
Pls do add the date along with the version for reference in the CHANGELOG.

Reply:
It's confusing so I don't add dates. This pr was opened yesterday. Yesterday is no longer representing the fix date as it's still open.